core/lib/dal/src/eth_sender_dal.rs

```diff
@@ -130,7 +139,8 @@ impl EthSenderDal<'_, '_> {
             FROM
                 eth_txs
             WHERE
-                id > (
+                from_addr IS NOT DISTINCT FROM $2
```

Comment:
wow this is a trick to get `null = null` return true right? Nice, wasn't aware. Maybe add a comment..

Reply:
+1, corrected